src/main/resources/frontend/src/views/StudentView/SubmissionInfo.vue

```diff
@@ -47,16 +48,16 @@ const { submission } = defineProps<{
   </div>
   <div class="container">
     <RubricItemView
-      v-if="submission.rubric.passoffTests"
```

Comment:
^Ditto

Vue's version of:
- For each rubric item (returned in our preferred order, ideally)
    - If the item value is non-null,
    - Render a `RubricItemView` with the item passed in properly